src/components/internals/create-select-styles.js

```diff
@@ -89,6 +88,29 @@ const menuListStyles = () => base => ({
   padding: '0',
   borderRadius: vars.borderRadiusInput,
   backgroundColor: vars.backgroundColorInputPristine,
+
+  '&::-webkit-scrollbar': {
```

Comment:
Where does all of this come from? 👀

Can you at least add some comments or something? Otherwise it looks like a huge hack (to me).

Reply:
The default setting for scrolling in Mac OSX is to only show scrollbars on scroll.

![Screen Shot 2019-03-29 at 2 55 19 PM](https://user-images.githubusercontent.com/2425013/55237560-baf56e80-5232-11e9-8620-02ae228c27ea.png)

The design team wants us to override this an always show the scrollbar. So we use CSS to "hack it" so that we can see the scrollbar.

Reply:
I added a comment, but it is a huge hack 😄

Reply:
Ah ok, so this is OS specific instead of browser specific?

Reply:
I appriciate the effort but am unsure what to think of this. It's almost like styling the scrollbar in the 90s ain't it 😄. Why do we overwrite a default which a user likely make on purpose on his OS?

Reply:
Maybe @Shecki can chime in and explain the reasoning from the design side.

Reply:
To answer the OS specific questions, yes this only effects Mac OS. On Windows, nothing changes.